phylogenetic/defaults/description.md

```diff
@@ -25,6 +32,9 @@ We curate sequence data and metadata from NCBI as the starting point for our ana
 * [data.nextstrain.org/files/workflows/lassa/all/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/lassa/all/sequences.fasta.zst)
 * [data.nextstrain.org/files/workflows/lassa/all/metadata.tsv.zst](https://data.nextstrain.org/files/workflows/lassa/all/metadata.tsv.zst)
 
+
+
+
```

Comment:
?

Reply:
Gah, I'll remove the extra lines, thanks 😝